### PR TITLE
Happy medium die-of-fate change

### PR DESCRIPTION
**Pull request by Ivniinvi**
## About The Pull Request
Attempts to reconcile #8654 and #8665 
Dice of Fate lose the 20th side when refreshed.

## Why It's Good For The Game
Stops excessive antag rolling while still allowing a niche, rare interaction to be carried out in a fun and engaging manner.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

die o fate
</details>

## Changelog
:cl:
balance: The Die of Fate loses its 20th face when being refreshed by an anomalous crystal.
/:cl:

### CODE REVIEW

**Comment by Rukofamicom** [comment]
This was the other option I liked more than removing the refreshing mechanics for die, but I still think the wizard chance deserves total removal from player hands.

File: code/modules/awaymissions/mission_code/Academy.dm [hunk 1]
```diff
@@ -172,6 +172,10 @@
 	var/used = FALSE
 	var/roll_in_progress = FALSE
 
+/obj/item/dice/d20/fate/proc/crystalrefresh()
+	sides = 19
+	visible_message("<span class='danger'>The 20th face of the [src] falls off and crumbles to dust!</span>")
```
Comment (Rukofamicom) [inline code/modules/awaymissions/mission_code/Academy.dm]:
```suggestion
/obj/item/dice/d20/fate/proc/crystalrefresh()
	if(sides == 20)
		sides = 19
		visible_message("<span class='danger'>The 20th face of the [src] falls off and crumbles to dust!</span>")
```

needs a check to ensure the 20th face doesn't crumble off more than once, no?

Comment (Ivniinvi) [reply code/modules/awaymissions/mission_code/Academy.dm]: That way wouldn't work, changed it in a way that does.

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
Code quality needs improving, istype should generally be avoided in situations like this in favour of proc calling or signals

**Comment by Pirill** [comment]
Best option in my opinion, and kind of funny too.

**Pull request closed without merging**